WebPhapp/client/src/App/pages/PrescriptionAdd.js

```diff
@@ -69,55 +69,141 @@ class PrescriptionAdd extends Component {
+                    <ul className="navbar-nav ml-lg-auto">
+                        <li className="nav-item">
+                            <a className="nav-link nav-link-icon" href="#">
+                                <i className="ni ni-favourite-28"></i>
+                                <span className="nav-link-inner--text d-lg-none">Discover</span>
+                            </a>
+                        </li>
+                        <li className="nav-item">
+                            <a className="nav-link nav-link-icon" href="#">
+                                <i className="ni ni-notification-70"></i>
+                                <span className="nav-link-inner--text d-lg-none">Profile</span>
+                            </a>
+                        </li>
+                        <li className="nav-item dropdown">
+                            <a className="nav-link nav-link-icon" href="#" id="navbar-default_dropdown_1" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                <i className="ni ni-settings-gear-65"></i>
+                                <span className="nav-link-inner--text d-lg-none">Settings</span>
+                            </a>
+                            <div className="dropdown-menu dropdown-menu-right" aria-labelledby="navbar-default_dropdown_1">
+                                <a className="dropdown-item" href="#">Action</a>
+                                <a className="dropdown-item" href="#">Another action</a>
+                                <div className="dropdown-divider"></div>
+                                <a className="dropdown-item" href="#">Something else here</a>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <br/>
 
-          <label>
-            Patient ID
-          </label>
-          <input
-            type="p"
-            value={this.state.patientID}
-            onChange={this.onKeyDownPatientID}
-          />
-
-          <label>
-            &nbsp; Drug&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.drugID}
-            onChange={this.onKeyDownDrugID}
-          />
-          <label>
-            &nbsp; Quantity&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.quantity}
-            onChange={this.onKeyDownQuantity}
-          />
-          <label>
-            &nbsp; Days Valid&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.daysValid}
-            onChange={this.onKeyDownDaysValid}
-          />
-          <label>
-            &nbsp; Dispensor ID&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.dispensorID}
-            onChange={this.onKeyDownDispensorID}
-          />
+        <form>
```

Comment:
We also have a field for `refills` which is an integer. Is this initially set to the same as `quantity`? if so, please confirm and then I can handle it on the backend later.

Reply:
@jacobkrantz no, i implemented the refills text field. good catch!